### DISCUSSION TITLE
前端界面不是很好看

### DISCUSSION THREAD

**Discussion by leookun** [admpub/nging#670] [General]
能否采用API和页面分离的形式，以便前端贡献代码

**Comment by admpub** [comment]
界面风格可以通过样式改变，不是通过前后端分离来实现的，它们没有必然联系。
把复杂的事情变简单是本项目的追求，其实我一直在观望前后端分离方案中的前端技术，之所以目前没有采用是因为目前的这种技术并没有把复杂的事情变简单：
1. 编译打包的加入增加了复杂度并延长了开发流程
2. 缓慢的首屏加载速度
3. 对多页面支持不友好
4. 网页模板风格不够丰富
5. 不能方便和细粒度的调整页面的每一个元素
在这些问题没有解决之前暂时不会考虑转过去。
如果你有更好的方案，欢迎推荐，重点就是不把简单的事情变复杂并且能够提高开发效率，即能够产生正向收益。
